### PR TITLE
Data migration: add doc for migration with binlog event filter and pt-ghost

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Data migration: add doc for migration with binlog event filter

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7480
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: data-migration/migrate-with-binlog-event-filter.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+title: Filter Binlog Events
+summary: Learn how to filter binlog events when migrating data
+---
+
+# Filter Binlog Events
+
+This document describes how to filter binlog events when you use DM to perform continuous incremental data replication. For the detailed replication instruction, refer to the following documents:
```
Comment (lance6716) [inline data-migration/migrate-with-binlog-event-filter.md:8]:
```suggestion
This document describes how to filter binlog events when you use DM to perform continuous incremental data replication. For the detailed replication instruction, refer to the following documents by scenarios:
```

File: data-migration/migrate-with-binlog-event-filter.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+- [Migrate MySQL Data of Less Than 1 TiB to TiDB](/data-migration/migrate-mysql-tidb-less-tb.md)
+- [Migrate MySQL Data of More Than 1 TiB to TiDB](/data-migration/migrate-mysql-tidb-above-tb.md)
+- [Migrate and Merge MySQL Shards of Less Than 1 TiB to TiDB](/data-migration/migrate-shared-mysql-tidb-less-tb.md)
+- [Migrate and Merge MySQL Shards of More Than 1 TiB to TiDB](/data-migration/migrate-shared-mysql-tidb-above-tb.md)
+
+## Configuration
+
+To use binlog event filter, add a `filter` when configuring the task configuration file of DM, as shown below:
```
Comment (lance6716) [inline data-migration/migrate-with-binlog-event-filter.md:17]: too many "configure" in this sentence, how about "add a `filter` when completing the task configuration file of DM"

File: data-migration/migrate-with-binlog-event-filter.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  | drop table      | DDL  | Drop table event      |
+  | truncate table  | DDL  | Truncate table event  |
+  | rename table    | DDL  | Rename table event    |
+  | drop index      | DDL  | Drop index event      |
+  | alter table     | DDL  | Alter table event     |
+
+- `sql-pattern`：Filter specified DDL SQL statements. The matching rule supports using a regular expression.
+- `action`: `Do` or `Ignore`
+
+    - `Do`: the allow list. A binlog event is filtered if meeting either of the following two conditions:
```
Comment (lance6716) [inline data-migration/migrate-with-binlog-event-filter.md:56]: seems the `Do` section is not same as Chinese version, do you deliberately refine it?

**Review by lance6716 — COMMENTED** [review]
will review later

File: data-migration/migrate-with-binlog-event-filter.md (hunk 2 above)
Comment (shichun-0415) [inline data-migration/migrate-with-binlog-event-filter.md:17]:
```suggestion
To use binlog event filter, add a `filter` to the task configuration file of DM, as shown below:
```

Comment (shichun-0415) [reply data-migration/migrate-with-binlog-event-filter.md:17]: @lance6716 PTAL

File: data-migration/migrate-with-binlog-event-filter.md (hunk 3 above)
Comment (shichun-0415) [reply data-migration/migrate-with-binlog-event-filter.md:56]: > seems the `Do` section is not same as Chinese version, do you deliberately refine it?

The previous Chinese is inaccurate and I have suggested Zhaoyang optimize it. The current English version is the translation of the previous Chinese. I will modify it based on the latest Chinese.

**Review by lance6716 — COMMENTED** [review]
> I will modify it based on the latest Chinese.

rest lgtm, and waiting above modification

File: data-migration/migrate-with-pt-ghost.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+title: Continuous Replication from Databases that Use gh-ost or pt-osc
+summary: Learn how to use DM to replicate incremental data from databases that use online DDL tools gh-ost or pt-osc
+---
+
+# Continuous Replication from Databases that Use gh-ost or pt-osc
+
+In production scenarios, table locking during DDL execution can block the reads or writes to and from the database to a certain extent. Therefore, online DDL tools are often used to execute DDLs to minimize the impact on reads and writes. Common DDL tools are [gh-ost](https://github.com/github/gh-ost) and [pt-osc](https://www.percona.com/doc/percona-toolkit/3.0/pt-online-schema-change.html).
```
Comment (shichun-0415) [inline data-migration/migrate-with-pt-ghost.md:8]:
```suggestion
In production scenarios, table locking during DDL execution can block the reads from or writes to the database to a certain extent. Therefore, online DDL tools are often used to execute DDLs to minimize the impact on reads and writes. Common DDL tools are [gh-ost](https://github.com/github/gh-ost) and [pt-osc](https://www.percona.com/doc/percona-toolkit/3.0/pt-online-schema-change.html).
```

File: data-migration/migrate-with-pt-ghost.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+For the detailed replication instruction, refer to the following documents by scenarios:
+
+- [Migrate MySQL Data of Less Than 1 TiB to TiDB](/data-migration/migrate-mysql-tidb-less-tb.md)
+- [Migrate MySQL Data of More Than 1 TiB to TiDB](/data-migration/migrate-mysql-tidb-above-tb.md)
+- [Migrate and Merge MySQL Shards of Less Than 1 TiB to TiDB](/data-migration/migrate-shared-mysql-tidb-less-tb.md)
+- [Migrate and Merge MySQL Shards of More Than 1 TiB to TiDB](/data-migration/migrate-shared-mysql-tidb-above-tb.md)
+
+## Enable online-ddl on DM
+
+When configuring the task configuration file of DM, set the global parameter `online-ddl` to `true`, as shown below:
```
Comment (shichun-0415) [inline data-migration/migrate-with-pt-ghost.md:21]:
```suggestion
In the task configuration file of DM, set the global parameter `online-ddl` to `true`, as shown below:
```

**Pull request closed without merging**